### ISSUE TITLE
Asynchronous evaluation

### ISSUE THREAD

**Issue by dkmiller** [Azure/azureml-examples#250]
## Long-running training script which submits parallel (asynchronous) evaluation logic

There's already 1P examples of this (e.g., the ELR team). But it's a good example to share with 3P.

**Comment by lostmygithubaccount** [comment]
it'd be great to have this to workaround the GHA 6 hour time limit too - spin off an action which runs hourly or so to check if the job has completed/failed

**Comment by lostmygithubaccount** [comment]
I think this is out of scope for this repository and may be better suited for a template repository

**Comment by dkmiller** [comment]
@lostmygithubaccount I disagree - the point of an examples repo is that you put examples there! Since Azure ML especially targets large enterprise customers, showcasing large enterprise scenarios like this one is valuable.
